wiki/Rules/id.md

```diff
@@ -1,67 +1,67 @@
 # Peraturan
 
-Peraturan-peraturan berikut ini merupakan pedoman dasar yang kami terapkan di dalam lingkungan osu! untuk menjaga agar komunitas osu! senantiasa menjadi tempat yang menyenangkan dan ramah bagi semua orang.
+Peraturan-peraturan berikut merupakan pedoman dasar kami dalam mewujudkan komunitas osu! yang ramah dan terbuka bagi semua kalangan.
 
-## Peraturan Komunitas
+## Peraturan komunitas
 
-1. **Setiap pemain hanya diperbolehkan untuk memiliki *SATU* akun.** Akun *pertama* yang kamu buat adalah satu-satunya akun osu! milikmu yang sah terhitung mulai dari kali pertama akun tersebut didaftarkan. Akun ini adalah KAMU. Bukan orang lain - bukan saudaramu, ibumu, kakakmu, temanmu - akun itu adalah milik KAMU. Jangan bagikan informasi kredensial akunmu dengan orang lain. Kamu tidak dibenarkan untuk membuat akun baru apabila dengan suatu alasan kamu kehilangan akses terhadap akun yang kamu miliki. Jagalah akunmu sebaik-baiknya.
-2. **Bermainlah dengan adil.** Penggunaan program-program ilegal yang dapat membuat permainan menjadi tidak adil dalam bentuk apapun sangat tidak diperbolehkan. Hal ini meliputi program makro, program yang meningkatkan akurasi bidikan kursor *(aim assist)*, program modifikasi skala waktu, dan lain sebagainya. Apabila kamu menyalahgunakan program-program tersebut, kamu akan sangat bersalah terhadap para pemain lainnya yang telah bermain adil!
-3. **Bersikap baiklah dengan saling menjaga rasa hormat antar satu sama lain.** Segala bentuk pelecehan dan hal-hal lainnya yang ditenggarai dapat merendahkan martabat orang lain tidak dibenarkan di dalam komunitas osu!. Kita di sini untuk bermain mengikuti irama musik, memukul drum, menangkap buah, dan menekan tuts keyboard - bukan untuk menjadi orang yang menyinggung perasaan orang lain.
+1. **Setiap pemain hanya diperbolehkan untuk memiliki *SATU* akun.** Akun *pertama* yang kamu buat adalah satu-satunya akun osu! milikmu yang sah terhitung mulai dari detik pertama akun tersebut didaftarkan. Akun ini adalah KAMU. Bukan orang lain — bukan saudaramu, ibumu, kakakmu, temanmu — akun itu adalah milik KAMU. Jangan bagikan informasi kredensial akunmu dengan orang lain. Kamu tidak dibenarkan untuk membuat akun baru apabila karena satu dan lain hal kamu tidak lagi dapat mengakses akun tersebut. Jagalah akunmu sebaik-baiknya.
```

Comment:
```suggestion
1. **Setiap pemain hanya diperbolehkan untuk memiliki *SATU* akun.** Akun *pertama* yang kamu buat adalah satu-satunya akun osu! milikmu yang sah terhitung mulai dari detik pertama akun tersebut didaftarkan. Akun ini adalah KAMU. Bukan orang lain—bukan saudaramu, ibumu, kakakmu, temanmu—akun itu adalah milik KAMU. Jangan bagikan informasi kredensial akunmu dengan orang lain. Kamu tidak dibenarkan untuk membuat akun baru apabila karena satu dan lain hal seperti kehilangan akses akun tersebut. Jagalah akunmu sebaik-baiknya.
```

Em-dash usage [PUEBI-Tanda Pisah](https://ivanlanin.github.io/puebi/tanda-baca/tanda-pisah/)

Reply:
o actually the original english text uses regular dash here so i think i'll just go with that

Reply:
it makes more sense to follow punctuation conventions of the language you translate to instead of copying the english version, which may be changed anyway (em dash instead of hyphen is the consensus on the english wiki)

Reply:
yeah the punctuation conventions in Indonesian are more or less the same w/ how it is in English (w/ space for regular dash character, w/o space for em dashes)

Pyrozen's suggestion above is more about the fact that the current article uses em dash (`—`) w/ space, which has since been fixed to regular dash w/ space follow the english text